client/views/create/create.html

```diff
@@ -74,6 +74,13 @@
 	                    <option value="hidden">Hidden</option>
```

Comment:
@singhsanket143 can you fix the code formatting of this file (indentation mostly)? It looks horrible ATM.

Reply:
Updated the formatting for this file.

Reply:
@singhsanket143 can you fix it in the whole file?

Reply:
@peter-hank I have updated the whole file. Please review it